### DISCUSSION TITLE
Matching TaxRef and deepfaune results

### DISCUSSION THREAD

**Discussion by MathildeNS** [naturalsolutions/ecoSecrets#61] [General]
Hello everyone,

We are currently trying to implement DeepFaune algorithm (https://www.deepfaune.cnrs.fr/) inside ecoSecrets. 

DeepFaune allows to classify the picture/video in several class (in french : bouquetin, cerf, chevreuil, chamois/isard, sanglier, loup, lynx, ours, blaireau, écureuil, genette, hérisson, lagomorphe, marmotte, mustélidé, ragondin, renard, chien, chat, chèvre, équidé, mouton, vache, micromammifère). 

In ecoSecrets, the user can annotate a media to tell which taxa are present on the media based on the french referential TaxRef.

Currently, the DeepFaune result is displayed as is on the annotation page. However, so that the user can benefit from DeepFaune's pre-processing of the media, we would like to be able to automatically enter an annotation based on the TaxRef. The user would then simply have to validate the pre-filled annotation.

To do that we are facing the question of the matching between TaxRef taxa and DeepFaune class, and which TaxRef taxon we should define by default to which DeepFaune class. Did some of you already use DeepFaune interface and how do you "translate" its prediction to standardize data to use it with other data in some SI like GeoNature for exemple ?

Thanks in advance for your answers.

_You can answer in french if it is easier for you_

**Comment by camillemonchicourt** [comment]
We think the simpliest would be to have a table in the database doing the correspondance between DeepFaune classes and Taxref cd_nom.
